modules/ossm-config-openshift-monitoring-kiali.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/observability/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-kiali_{context}"]
+= Configuring OpenShift Monitoring with Kiali
+:context: ossm-config-openshift-monitoring-kiali
+
+// TP 1 content.
+// Possible file name may change
+// Possible assembly file may change
+// Assemblies, topic map info needs to be worked out still for 3.0.
+// Possible content will change.
+
+The following steps show how to integrate the {KialiProduct} with `user-workload monitoring`.
+
+.Prerequisites
+
+* {SMProductName} is installed.
+
+* `user-workload monitoring` is enabled. See "Enabling user-defined projects".
+
+* {KialiProduct} 1.89 is installed.
+
+.Procedure
+
+. Create a `ClusterRoleBinding` resource for Kiali:
++
+.Example `ClusterRoleBinding` configuration
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kiali-monitoring-rbac
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-monitoring-view
+subjects:
+- kind: ServiceAccount
+  name: kiali-service-account
+  namespace: istio-system
+----
+
+. Create a Kiali resource and point it to your Istio instance:
++
+.Example Kiali resource configuration
+[source,yaml]
+----
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali-user-workload-monitoring
+  namespace: istio-system
+spec:
+  external_services:
+    prometheus:
+      auth:
+        type: bearer
+        use_kiali_token: true
+      thanos_proxy:
+        enabled: true
+      url: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
+----
+
+. When the Kiali resource is ready, get the Kiali URL from the `Route` using the following command:
++
+[source,terminal]
+----
+echo "https://$(oc get routes -n istio-system kiali -o jsonpath='{.spec.host}')"
```

Comment:
Also just checking, disregard if this isn't the case - since there's an oc command seemingly buried in here, does the user need to have the oc CLI installed to do this? If so, I would add "You have installed the OpenShift CLI (`oc`)." as a prereq to this procedure.